### ISSUE TITLE
[Proposal] Import from document editor

### ISSUE THREAD

**Issue by ocdtrekkie** [audreyt/ethercalc#105]
One thing both Excel and Google Sheets have in common notably, is the fact that you can create a spreadsheet document outright, and then import data into it. Whereas EtherCalc's current flow is to either import a document or create a document.

The implementation I am using inside Sandstorm, basically has a single new document option, so it goes directly to the document editor. I'm dubious of my own ability to implement this, but it'd be great if there was a way to access the data import functionality from inside the spreadsheet editor. With, I assume, a warning about the incoming data overwrite if you proceed.

A use case for this that would also apply to ethercalc.org was if you had a shared EtherCalc URL for sharing data with someone, but you have been editing via a desktop app, like Excel, and wanted to update the entire sheet, at the same URL with what you've been working on offline.

**Comment by audreyt** [comment]
Re-opening for Sandstorm file picker and server-side importing option (for clients that can't afford to convert XLSX in JS).

**Comment by OliverHi** [comment]
I have a similiar (or the same?) question: I want to import an excel file into an ethercalc grain in sandstorm. New one or existing one doesn't really matter for me.
Is that possible yet? I couldn't find out how.

**Comment by ocdtrekkie** [comment]
@OliverHi You can! Drag the file you want to import onto the "Sheet1" tab on the bottom of your EtherCalc window! It'll overwrite the existing sheet contents.

**Comment by OliverHi** [comment]
Ah thank you a lot!! Working fine for me :) 
I would still suggest to make that more obvious (as well as the download button).

**Comment by ocdtrekkie** [comment]
Yeah, my personal opinion is it makes sense to have a menu option somewhere up top that lets you open a file selection dialog and stuff. But I'm glad I was able to help you figure out how to use the feature that exists as it does now!
